diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 6bf82a39..a924895f 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -211,13 +211,13 @@ endif()
 
 # TODO : We should not include include/exiv2 but only include !!!
 target_include_directories(exiv2lib PUBLIC
-    $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include/exiv2>
-    $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include>
+    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include/exiv2>
+    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
     $<INSTALL_INTERFACE:include>
 )
 
 target_include_directories(exiv2lib_int PUBLIC
-    $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include/exiv2>
+    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include/exiv2>
 )
 
 if (EXIV2_ENABLE_WEBREADY)
diff --git a/xmpsdk/CMakeLists.txt b/xmpsdk/CMakeLists.txt
index ff8fd8bb..68913ded 100644
--- a/xmpsdk/CMakeLists.txt
+++ b/xmpsdk/CMakeLists.txt
@@ -37,7 +37,7 @@ target_link_libraries(xmp
 
 target_include_directories(xmp
     PUBLIC 
-        $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/xmpsdk/include>
+        $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/xmpsdk/include>
     PRIVATE 
         ${EXPAT_INCLUDE_DIR}
 )
